templ_attr.cif

```diff
@@ -612,21 +617,74 @@ save_aniso_bij_su
      save_
 
 
+save_aniso_betaij
+
+    _definition.update           2023-02-16
+    _description.text
+;
+     These are the standard unitless anisotropic atomic displacement components
+     which appear in the structure factor term:
+
+         T = exp{-1 sum~i~ [ sum~j~ (\b^ij^ h~i~ h~j~) ] }
+
+     h = the Miller indices
+
+     The unique elements of the real symmetric matrix are entered by row.
+
+     The IUCr Commission on Nomenclature recommends against the use
+     of \b for reporting atomic displacement parameters; U is preferred [1].
+
+     Note that U^ij^ = \b^ij^/(2 pi^2^ a*~i~ a*~j~) = B^ij^/(8 pi^2^) [1].
+
+     [1] Trueblood, K. N. et al. (1996). Acta Crystallogr. A52(5), 770-781.
```

Comment:
Given that the Unicode branch is now merged in, we might want to update the text here as well to contain proper Unicode symbols.